CHANGELOG.md

```diff
@@ -23,6 +23,11 @@
 
 🔧 Fixes:
 
+- Apply `display:block` to `.govuk-main-wrapper`
```

Comment:
If you're intending to add a line-break here, you need to leave a blank line. At the minute it wraps onto one line and reads like this:

> Apply `display:block` to `.govuk-main-wrapper` In IE11 `main` element is set to `display:inline` so padding an margins aren't applied.

Also, should be *and* instead of *an*.

Reply:
updated